drink/src/runtime/mod.rs

```diff
@@ -20,33 +21,20 @@ pub use pallet_balances;
 pub use pallet_contracts;
 
 /// A runtime to use.
-///
-/// Must contain at least system, balances and contracts pallets.
-pub trait Runtime:
-    frame_system::Config
-    + pallet_timestamp::Config
-    + pallet_balances::Config<Balance = u128>
-    + pallet_contracts::Config<Currency = pallet_balances::Pallet<Self>>
-{
+pub trait Runtime: frame_system::Config {
     /// Initialize the storage at the genesis block.
-    fn initialize_storage(_storage: &mut Storage) -> Result<(), String> {
-        Ok(())
```

Comment:
why the default implementation is bad in this trait? I can imagine runtimes with empty work needed for some phases

Reply:
Ah I can put that back, I felt that this could be removed since for now all implementations need to implement these fn explicitly

Reply:
yeah, I think we can restore it